src/main/java/com/woowacourse/tecobrary/user/command/util/UserJwtVoMapper.java

```diff
@@ -5,13 +5,12 @@
 
 public class UserJwtVoMapper {
 
-    public static UserJwtInfoVo map(final User user) {
-        return new UserJwtInfoVo(
-                user.getId(),
-                user.getUserEmail(),
-                user.getUserName(),
-                user.getUserAvatarUrl(),
-                user.getAuthorization()
-        );
+    public static UserJwtInfoVo toDomain(final User user) {
```

Comment:
```suggestion
    public static UserJwtInfoVo toVo(final User user) {
```

혹은 Vo 의 네이밍을 Dto 로 변경하고

```suggestion
    public static UserJwtInfoDto toDto(final User user) {
```

로 하는 것이 어떨까용

Reply:
- UserJwtInfoVo를 필드로 갖고 있는 GithubApiResponseVo가 DTO라고 생각해서 GithubApiResponseVo이름을 GithubApiResponseDto로 변경했습니다.
- UserJwtInfoVo는 VO가 맞는 것 같아요. 그래서 mapper 클래스의 메서드명을 toVo로 변경했습니다.